Assets/Scripts/ChallengeManagerScript.cs

```diff
@@ -166,6 +166,11 @@ public void UpdateChallengeText(string newText)
         challengeTitle.GetComponent<Text>().text = newText;
         // TODO: Make a helper function to format the challenge number string
         challengeNumber.GetComponent<Text>().text = "CHALLENGE 0" + (DataManagerScript.challengeType + 1);
+        // Special case for challenge 10
+        if (DataManagerScript.challengeType == 9)
+        {
+            challengeNumber.GetComponent<Text>().text = "CHALLENGE 10";
+        }
```

Comment:
If you wanted to make this more extensible, I think you could pad a zero like: 
```
int value = DataManagerScript.challengeType;
int paddedLength= 2;
int decimalLength = paddedLength - value.ToString("D").Length;
string paddedString = value.ToString("D" + decimalLength.ToString());
```
I adapted this [from an example](https://docs.microsoft.com/en-us/dotnet/standard/base-types/how-to-pad-a-number-with-leading-zeros) so I'm not sure if it works as is, but something like that.

Reply:
Cool, made this change -